en/components/exporter_csv.md

```diff
@@ -0,0 +1,204 @@
+
+Here is the code which will execute the export process in the component's typescript file:
+
+```typescript
+// component.ts
+
+...
+import { IgxCsvExporterService, IgxCsvExporterOptions, CsvFileTypes } from "igniteui-angular/services/index";
+...
+
+public localData = [
+  { Name: "Eric Ridley", Age: "26" },
+  { Name: "Alanis Brook", Age: "22" },
+  { Name: "Jonathan Morris", Age: "23" }
+];
+
+constructor(private csvExportService: IgxCsvExporterService) {
+}
+
+public exportButtonHandler() {
+  this.csvExportService.exportData(this.localData, new IgxCsvExporterOptions("ExportedDataFile"), CsvFileTypes.CSV);
+}
+
+```
+
+If all went well, you should see an export button. When pressed, it will trigger the export process and the browser will download a file named "ExportedDataFile.csv" which contains the data from the `localData` array in CSV format. 
+
+
+### Exporting IgxGrid's Data
+
+The CSV Exporter service can also export data in CSV format from an `IgxGrid`. The only difference if that you need to invoke the 
+`IgxCsvExporterService`'s `export` method and pass the `IgxGrid` as first argument.
+
+Here is an example:
+
+```html
+<igx-grid #igxGrid1 [data]="localData" [autoGenerate]="true"></igx-grid>
+<button (click)="exportButtonHandler()">Export IgxGrid</button>
+```
+
+```typescript
+// component.ts
+
+...
+import { IgxCsvExporterService, IgxCsvExporterOptions, CsvFileTypes } from "igniteui-angular/services/index";
+import { IgxGridComponent } from "igniteui-angular/grid/grid.component";
+...
+
+@ViewChild("igxGrid1") public igxGrid1: IgxGridComponent;
+
+public localData = [
+  { Name: "Eric Ridley", Age: "26" },
+  { Name: "Alanis Brook", Age: "22" },
+  { Name: "Jonathan Morris", Age: "23" }
+];
+
+constructor(private csvExportService: IgxCsvExporterService) {
+}
+
+public exportButtonHandler() {
+  this.csvExportService.export(this.igxGrid1, new IgxCsvExporterOptions("ExportedDataFile", CsvFileTypes.CSV));
+}
+
+```
+
+<div class="sample-container loading" style="height: 300px;">
+    <iframe id="csv-export-sample-iframe2" src="{environment:demosBaseUrl}/export-csv-sample-1"
+        width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+<button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="csv-export-sample-iframe2"
+    data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+
+### Customizing the Exported Format
+
+The CSV Exporter supports several types of exporting formats. The export format may be specified:
+* as second argument of the `IgxCsvExporterOptions` objects's constructor
```

Comment:
Should be "as **a** second argument of the `IgxCsvExporterOptions` **object's** constructor